alex/applications/PublicTransportInfoCS/nlg_templates.cfg

```diff
@@ -913,4 +913,24 @@ templates = {
 
     'apology()&inform(in_city={in_city})':
         "Omlouvám se, počasí pro obec {in_city} nemám k dispozici.",
+
+    #
+    # Platform info related templates.
+    #
+    'request(to_stop)&request(train_name)':
+        "A jakým směrem chcete jet? Nebo jak se jmenuje vlak?",
+    'inform(platform=none)&inform(track=none)&inform(direction={direction})':
+        "Ještě nění známo odkud to směrem {direction} jede.",
```

Comment:
This word order sounds a bit unnatural to me – I'd use "odkud to jede směrem {...}".